common/config/rush/common-versions.json

```diff
@@ -62,6 +62,10 @@
     "@opentelemetry/api": ["^0.14.0"],
     // Monitor: Allow node 10 types until Timeout / Timer conflict is resolved in OpenTelemetry repo
     // TODO: remove when released https://github.com/open-telemetry/opentelemetry-js/pull/1352
-    "@types/node": ["^10.0.0"]
+    "@types/node": ["^10.0.0"],
+
+    // @azure/test-utils-perfstress should depend on lowest version of @azure/core-http for maximum compatibility, allowing test
+    // projects to choose a higher version if desired.
+    "@azure/core-http": ["^1.0.0"]
```

Comment:
The tests would still depend on the latest core-http versions as set at the SDK level(example: storage-blob), what is the use of this then?

Reply:
The tests need the ability to target older package versions, say `@azure/storage-blob@12.0.0` with `@azure/core-http@1.0.0`.  Currently, the tests are hard-coded to only run against the SDK sources in the same branch as the test, but this needs to change.

Without this change, tests would be unable to target versions of `@azure/core-http` older than `1.2.0`.